SDPi_Supplement/asciidoc/volume3/tf3-ch-8.2.8-biceps-content.adoc

```diff
@@ -4,11 +4,36 @@ NOTE:  This file contains the content that profiles BICEPS semantics for use in
 
 // 8.2.8
 [sdpi_offset=8]
-===== SDC/BICEPS semantic content
+==== SDC/BICEPS semantic content
 
 // 8.2.8.1
 ===== SDC/BICEPS Content Module
 
+====== BICEPS Extension Model
+
+IEEE 11073-10207 provides a flexible extension model.
+This enables manufacturers to add information that does not fit into the defined structure.
+Therefore, nearly every element in the participant and message model contains an extension point (`ext:Extension`).
+However, the fundamental rules are: Do not use a proprietary extension if the purpose of the extension can be expressed by (in the specified order)
+
+. the given participant and message models,
+. a standardized coded value,
+. a standardized extension, or
+. a private coded value.
+
+
+The borders of using extension are given by:
+
+* <<ieee_11073_10207_2017>> Clause 8, especially R0109, and
+* <<ieee_11073_10700_2022>> Clause 6.7
+
+There are two kinds of extensions, those, the <<actor_somds_consumer>> must understand to perform system functions safely with the <<actor_somds_provider>>, and those the <<actor_somds_consumer>> does not have to understand.
+This is indicated by the Boolean attribute `@ext:MustUnderstand`.
+Consequently, a manufacturer of a <<actor_somds_provider>> should be careful with defining extension with `@ext:MustUnderstand = true`, as this potentially reduces interoperability.
+
+There are standardized extensions, like those that are specified in the PKP standards or this profile, which can be considered common knowledge and therefore do not break interoperability between different manufacturers.
+Consequently, these extensions can be seen as common knowledge and do not hinder interoperability between different manufacturers.
```

Comment:
Remove this sentence since all it expresses is already explained in the previous sentence.

Reply:
Is this extension model in 8.2.8.8?  (the one allocated in the Word draft section 8.2.8)